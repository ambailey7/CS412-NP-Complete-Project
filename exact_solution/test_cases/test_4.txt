12 66
0 1 483.854
0 2 790.1
0 3 293.291
0 4 488.88
0 5 247.452
0 6 631.812
0 7 266.48
0 8 709.327
0 9 723.637
0 10 968.735
0 11 37.651
1 2 418.081
1 3 134.265
1 4 316.868
1 5 294.118
1 6 419.82
1 7 602.469
1 8 606.157
1 9 77.829
1 10 830.903
1 11 876.768
2 3 898.761
2 4 882.588
2 5 107.464
2 6 153.91
2 7 9.408
2 8 820.214
2 9 203.56
2 10 675.745
2 11 84.107
3 4 367.949
3 5 859.902
3 6 204.45
3 7 994.505
3 8 112.203
3 9 334.94
3 10 710.192
3 11 981.913
4 5 19.095
4 6 286.928
4 7 182.519
4 8 196.609
4 9 163.973
4 10 541.754
4 11 555.969
5 6 868.744
5 7 4.028
5 8 917.701
5 9 994.748
5 10 551.794
5 11 618.584
6 7 878.471
6 8 336.922
6 9 568.823
6 10 532.796
6 11 621.149
7 8 999.902
7 9 724.686
7 10 683.231
7 11 999.849
8 9 983.353
8 10 232.843
8 11 319.367
9 10 108.323
9 11 907.526
10 11 788.84