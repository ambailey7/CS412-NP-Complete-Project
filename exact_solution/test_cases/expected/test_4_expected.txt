1519.59
4 5 7 0 11 2 6 3 1 9 10 8 4